### ISSUE TITLE
Essi - Test IIIF Print

### ISSUE THREAD

**Issue by kirkkwang** [scientist-softserv/iiif_print#155]
# Summary

After completing #150, #151, #152, #153, #154, test the full functionality of IIIF Print on Essi.

# Acceptance Criteria

- [ ] As a user, I can view parent and child images grouped together in a manifest
- [ ] As a user, when I upload an import, PDFs are split into component pages, OCR'd and derivative files are created for each page via a configurable job
- [ ] As a user, I can configure the derivative generation between the following options (JP2, PDF, txt, TIFF)
- [ ] The gem is compatible with v2 and v3 manifest
- [ ] The gem is compatible with dynamic metadata profiles
- [ ] As a user, when I select a work type, it's child works show up in the viewer under the parent item
- [ ] As a user, I can easily navigate between child and parent works in the Universal Viewer and they appear in a logical order
- [ ] As a user, I can see all of the metadata properties of a parent and child work in the Universal Viewer
- [ ] As a user, I can zoom in on the the viewer
- [ ] As a user, I can use paginated navigation
- [ ] As a user, I can search the Universal Viewer for child or parent keywords
- [ ] As a user, I can see my child or parent keyword searches highlighted on the show page
- [ ] As a user, when I click on a viewer facet, it will redirect me to a search
- [ ] As a user, I can download the text of a work in a PDF format
- [ ] As a user, I can click a URL on a work show page to access its manifest in a JSON format
- [ ] As a user, I can click a URL on a work show page to access its API endpoint

# Notes

https://docs.google.com/spreadsheets/d/1V3DoknWcmGQwc5StU1zBQxpGRQ7ttctwxXpbfxJvtsw/edit#gid=882540390

**Comment by ShanaLMoore** [comment]
depends on #144

**Comment by jillpe** [comment]
Related to [essi #10](https://github.com/scientist-softserv/essi/issues/10)

**Comment by jillpe** [comment]
closed - duplicate of essi #10
